db/postgres/connection.go

```diff
@@ -423,7 +456,7 @@ func (d *DB) RollbackTransaction() error {
 // https://www.postgresql.org/docs/9.2/static/sql-set.html
 func (d *DB) Set(set string) error {
 	if d.tx == nil {
-		return nil
+		return errors.New("cannot set a local variable in an ongoing transaction without a transaction")
```

Comment:
please make this into an error in the errors lib (this facilitates error checking)